spec/features/expenses/deletes_expense.rb

```diff
@@ -0,0 +1,21 @@
+require "rails_helper"
+
+feature "delete expense" do
+  scenario "successfully" do
+    user = create(:user)
+    create(:budget, user: user)
+    log_in_as user
+    expense = create(:expense)
+
+    visit edit_expense_path(expense)
+
+    click_on I18n.t("expenses.edit.delete")
+
+    expect(page).to have_text(I18n.t("expenses.new.title"))
+    expect(page).
+      to have_text(I18n.t(
+        "flash.actions.destroy.notice",
+        resource_name: "Expense"
+      ))
```

Comment:
Maybe add an assertion to check that the expense name is not on the page?

Reply:
Done in sha: 0648c4a